app/templates/suppliers/edit_what_buyers_will_see.html

```diff
@@ -0,0 +1,106 @@
+      },
+      {
+        "link": url_for(".dashboard"),
+        "label": "Your account"
+      },
+      {
+        "link": url_for(".supplier_details"),
+        "label": "Company details"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+{% if error %}
+<div class="validation-masthead" aria-labelledby="validation-masthead-heading">
+  <h3 class="validation-masthead-heading" id="validation-masthead-heading">
+    {{ error }}
+  </h3>
+</div>
+{% endif %}
+{% if supplier_form.errors or contact_form.errors %}
+    <div class="validation-masthead" aria-labelledby="validation-masthead-heading">
+        <h3 class="validation-masthead-heading" id="validation-masthead-heading">
+            There was a problem with the details you gave for:
+        </h3>
+        {% if supplier_form.errors %}
+        <ul>
+        {% for field_name, field_errors in supplier_form.errors|dictsort %}
+          {% for error in field_errors %}
+            <li><a href="#{{ field_name }}" class="validation-masthead-link">{{ supplier_form[field_name].label.text }}</a>
+          {% endfor %}
+        {% endfor %}
+        </ul>
+        {% endif %}
+
+        {% if contact_form.errors %}
+        <ul>
+        {% for field_name, field_errors in contact_form.errors|dictsort %}
+          {% for error in field_errors %}
+          <li><a href="#contact_{{ field_name }}" class="validation-masthead-link">{{ contact_form[field_name].label.text }}</a></li>
+          {% endfor %}
+        {% endfor %}
+        </ul>
+        {% endif %}
+    </div>
+{% endif %}
+
+  <div class="grid-row">
+    <div class="column-one-whole">
+      {% with
+        heading = "What buyers will see"
+      %}
+        {% include 'toolkit/page-heading.html' %}
+      {% endwith %}
+
+      <div class="dmspeak">
+        <p>This information will be visible on the Digital Marketplace.</p>
+        <p>You can change it at any time.</p>
+      </div>
+    </div>
+  </div>
+
+  <form action="{{ url_for('.edit_what_buyers_will_see') }}" method="post" enctype="multipart/form-data">
+
+  <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      {{ forms.question_textbox('contactName', 'Contact name', contact_form.contactName.data, 'Primary contact', errors=contact_form.contactName.errors) }}
+      {{ forms.question_textbox('email', 'Contact email', contact_form.email.data, errors=contact_form.email.errors) }}
+      {{ forms.question_textbox('phoneNumber', 'Contact phone number', contact_form.phoneNumber.data, errors=contact_form.phoneNumber.errors) }}
+
+
+      {{ forms.question_textarea('description', 'Supplier summary', supplier_form.description.data, '50 words max', errors=supplier_form.description.errors, max_length_in_words=50) }}
+    </div>
+  </div>
+
+  {%
+    with
+    type = "save",
+    label = "Save and return"
```

Comment:
Label on the prototype is "Continue". This sounds better but maybe worth checking?

Reply:
I asked @constancecerf about this and other one, will see what she says :). Thanks for spotting! <3

Reply:
So this should be "Save and continue", I'll change it in a minute